test/units/_internal/templating/test_lazy_containers.py

```diff
@@ -466,8 +466,8 @@ def test_range_templating():
     templar = TemplateEngine()
 
     # ensure that an insanely large range is not listified
-    assert templar.evaluate_conditional(TRUST.tag("range(1000000000000) | type_debug == 'range'"))
-    assert isinstance(templar.template(TRUST.tag("{{ range(1000000000000) | random }}")), int)
+    assert templar.evaluate_conditional(TRUST.tag("range(2147483647) | type_debug == 'range'"))
+    assert isinstance(templar.template(TRUST.tag("{{ range(2147483647) | random }}")), int)
```

Comment:
Lets keep the larger value on systems that can handle it, and limit it only for 32-bit systems:

```suggestion
    large_value = min(1000000000000, sys.maxsize - 1)  # ensure we don't exceed ssize_t on 32-bit systems
    assert templar.evaluate_conditional(TRUST.tag("range(large_value) | type_debug == 'range'"))
    assert isinstance(templar.template(TRUST.tag("{{ range(large_value) | random }}")), int)
```

Reply:
Also works for me. I tried just using `sys.maxsize` in PR #85006 and that failed. I assumed that was just a scoping issue and didn't dig any further.

Reply:
Yeah, you'll also need to import `sys` for this to work.

Reply:
Yeah, that was part of the PR. pylint complained about unused import, and the assert failed, though I only skimmed the log in question.

Reply:
Oh, I just realized my suggested edit was incorrect. The `large_value` variable isn't exposed in the template, so it won't evaluate properly.

Reply:
@leegarrett If you don't mind, I can push the necessary changes to this PR.

Reply:
Sure, go ahead.